externals/CMake/FindNETCDF.cmake

```diff
@@ -19,7 +19,7 @@ find_library(NetcdfF_LIBRARY
 find_path(Netcdf_NC_CONFIG_BIN
           NAMES nc-config
           HINTS ${Netcdf_INCLUDE_DIR}/../bin
-          NO_SYSTEM_ENVIRONMENT_PATH NO_CMAKE_SYSTEM_PATH)
+          NO_CMAKE_SYSTEM_PATH)
```

Comment:
This could lead to errors if a user has NETCDF_PATH set incorrectly in the environment.  Currently we are requiring it to be explicitly set in config_machines.xml or config_compilers.xml.